js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetNodeStatusContent.tsx

```diff
@@ -62,7 +62,7 @@ export function buildAssetNodeStatusContent({
   liveData,
   expanded,
 }: StatusContentArgs) {
-  return definition.isSource
+  return definition.isObservable
     ? _buildSourceAssetNodeStatusContent({
```

Comment:
should this fn get renamed accordingly?

Reply:
Ahh yes, going to do this in a small follow-up to keep this PR the logic changes only